static/app/components/suggestProjectCTA.tsx

```diff
@@ -1,223 +0,0 @@
-      ) ?? ''
-    );
-  }
-
-  // check our projects to see if there is a mobile project
-  get hasMobileProject() {
-    return this.props.projects.some(project =>
-      MOBILE_PLATFORMS.includes(project.platform || '')
-    );
-  }
-
-  // returns true if the current event is mobile from the user agent
-  // or if we found a mobile event with the API
-  get hasMobileEvent() {
-    const {mobileEventResult} = this.state;
-    return !!this.matchedUserAgentString || !!mobileEventResult;
-  }
-
-  /**
-   * conditions to show prompt:
-   * 1. Have a mobile event
-   * 2. No mobile project
-   * 3. CTA is not dimissed
-   * 4. We've loaded the data from the backend for the prompt
-   */
-  get showCTA() {
-    const {loaded, isDismissed} = this.state;
-
-    return !!(this.hasMobileEvent && !this.hasMobileProject && !isDismissed && loaded);
-  }
-
-  async fetchData() {
-    // no need to catch error since we have error boundary wrapping
-    const [isDismissed, mobileEventResult] = await Promise.all([
-      this.checkMobilePrompt(),
-      this.checkOrgHasMobileEvent(),
-    ]);
-
-    // set the new state
-    this.setState(
-      {
-        isDismissed,
-        mobileEventResult,
-        loaded: true,
-      },
-      () => {
-        const matchedUserAgentString = this.matchedUserAgentString;
-        if (this.showCTA) {
-          // now record the results
-          trackAdvancedAnalyticsEvent(
-            'growth.show_mobile_prompt_banner',
-            {
-              matchedUserAgentString,
-              mobileEventBrowserName: mobileEventResult?.browserName || '',
-              mobileEventClientOsName: mobileEventResult?.clientOsName || '',
-              organization: this.props.organization,
-            },
-            {startSession: true}
-          );
-        }
-      }
-    );
-  }
-
-  async checkOrgHasMobileEvent(): Promise<MobileEventResult> {
-    const {api, organization} = this.props;
-    return api.requestPromise(
-      `/organizations/${organization.slug}/has-mobile-app-events/`,
-      {
-        query: {
-          userAgents: MOBILE_USER_AGENTS,
-        },
-      }
-    );
-  }
-
-  async checkMobilePrompt() {
-    const {api, organization} = this.props;
-
-    // check our prompt backend
-    const promptData = await promptsCheck(api, {
-      organizationId: organization.id,
-      feature: 'suggest_mobile_project',
-    });
-    return promptIsDismissed(promptData);
-  }
-
-  handleCTAClose = () => {
-    const {api, organization} = this.props;
-    trackAdvancedAnalyticsEvent('growth.dismissed_mobile_prompt_banner', {
-      matchedUserAgentString: this.matchedUserAgentString,
-      organization,
-    });
-
-    promptsUpdate(api, {
-      organizationId: organization.id,
-      feature: 'suggest_mobile_project',
-      status: 'dismissed',
-    });
-
-    this.setState({isDismissed: true});
-  };
-
-  openModal = () => {
-    trackAdvancedAnalyticsEvent('growth.opened_mobile_project_suggest_modal', {
```

Comment:
@Neo-Zhixing we can remove the analytics events as well

Reply:
good point